.mergify.yml

```diff
@@ -143,6 +143,7 @@ pull_request_rules:
           For such, you'll need to label your PR with:
           * The upcoming major version of the Elastic Stack
           * The upcoming minor version of the Elastic Stack (if you're not pushing a breaking change)
+
```

Comment:
cosmetic change so the new line will help with a nicer UI, otherwise it's listed instead